Elevel/Application/Features/AuditionFeatures/GetAuditionListQuery.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoMapper;
+using Elevel.Application.Interfaces;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Elevel.Application.Pagination;
+using AutoMapper.QueryableExtensions;
+
+namespace Elevel.Application.Features.AuditionFeatures
+{
+    public class GetAuditionListQuery
+    {
+        public class Request : PagedQueryBase, IRequest<Response>
+        {
+        }
```

Comment:
Don't you need any parameters for searching some exact type of Question?

Reply:
@andreyrogov8 Let's better divide the tasks into smaller parts, while I suggest leaving it that way.